src/exchange/tradingPair/TradingPairView.scss

```diff
@@ -104,37 +109,70 @@
 
 
 
-// ------------ Pair view -----------------
+// ------------ Active pair view -----------------
 
-.pairView {
+.activePairView {
     display: flex;
     justify-content: space-between;
     align-items: center;
     padding: 12px 26px 12px 12px;
     width: 100%;
 }
 
-.pairViewCurrency {
+.activePairViewIcon {
+    height: 26px;
+    width: 26px;
+    margin-right: 20px;
+    display: flex;
+    align-items: center;
+}
+
+.activePairViewCurrency {
     flex-basis: 50px;
     flex-grow: 1;
 }
 
-.pairViewIcon {
-  height: 26px;
-  width: 26px;
-  margin-right: 20px;
-  display: flex;
-  align-items: center;
+// ------------ Pair view -----------------
+
+.pairView {
+    display: table-row;
+    background-color: #35353E;
+    &:hover {
+        background-color: #393942;
+    }
+    font-size: 14px;
+    & > * {
+      display: table-cell;
```

Comment:
`inline-flex` with `align-items:center` ?

Reply:
Not with the table layout

Reply:
Well you don't need the table layout. Because you have couple of things in a line, don;t make them tabular data. Also you are using a list with list-items for the dropdown and then you have `display:table`

Reply:
I need the table because I cannot predict the length of the prices and percentages.

Reply:
Not quite sure how table is helping you with this one ...